include/gepetto/gui/mainwindow.hh

```diff
@@ -17,6 +17,7 @@
 #include <gepetto/gui/dialog/pluginmanagerdialog.hh>
 
 #include <gepetto/gui/settings.hh>
+#include <gepetto/gui/shortcut-factory.hh>
```

Comment:
To reduce compilation time, please:
- predefine `ShortcutFactory` in `gepetto/gui/fwd.hh`
- move `#include <gepetto/gui/shortcut-factory.hh>` to `mainwindow.cc`